src/molecules/Table/Table.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react';
+import { useTable, useSortBy, Column } from 'react-table';
+
+import { Icon } from 'atoms';
+import { StyledTable } from './Table.style';
+
+type Data = {
+  [key: string]: React.ReactNode;
+};
+
+interface Props {
+  theme?: 'default' | 'condensed';
+  columns: Array<Column<Data>>;
+  data: Array<Data>;
+}
+
+export type TableProps = Props & React.HTMLAttributes<any>;
```

Comment:
```suggestion
export type TableProps = Props & React.TableHTMLAttributes<HTMLTableElement>;
```

With this change, now it would make sense to pass `...otherProps` to the `<StyledTable>` because it will include any html table attributes as possible props, although MDN says they are all deprecated. Probably still worth doing just in case.